index.html

```diff
@@ -175,39 +192,45 @@ <h1>About me</h1>
       <button class="get-myresume-btn" type='button'>Get my resume</button>
     </div>
     <div class="competence">
-      <ul class="competence-list">
-        <li class="list-title languages">
-          Languages
-        </li>
-      </ul>
-      <ul class="competence-list">
-        <li class="competence-item">Javascript</li>
-        <li class="competence-item">Ruby</li>
-        <li class="competence-item">Css</li>
-        <li class="competence-item">Html</li>
-      </ul>
-      <ul class="competence-list">
-        <li class="list-title frameworks">
-          Frameworks
-        </li>
-      </ul>
-      <ul class="competence-list">
-        <li class="competence-item">Bootstrap</li>
-        <li class="competence-item">React</li>
-        <li class="competence-item">Gatsby</li>
-        <li class="competence-item">Laravel</li>
-      </ul>
-      <ul class="competence-list">
-        <li class="list-title skills">
-          Skills
-        </li>
-      </ul>
-      <ul class="competence-list">
-        <li class="competence-item">Github</li>
-        <li class="competence-item">Codepen</li>
-        <li class="competence-item">Codesandbox</li>
-        <li class="competence-item">Replit</li>
-      </ul>
+      <div class="languajes-div">
+        <ul class="competence-list">
+          <li class="list-title languages">
+            Languages
+          </li>
+        </ul>
+        <ul class="competence-list">
+          <li class="competence-item">Javascript</li>
+          <li class="competence-item">Ruby</li>
+          <li class="competence-item">Css</li>
+          <li class="competence-item">Html</li>
+        </ul>
+      </div>
+      <div class="frameworks-div">
+        <ul class="competence-list">
+          <li class="list-title frameworks">
+            Frameworks
+          </li>
+        </ul>
+        <ul class="competence-list">
+          <li class="competence-item">Bootstrap</li>
+          <li class="competence-item">React</li>
+          <li class="competence-item">Gatsby</li>
+          <li class="competence-item">Laravel</li>
+        </ul>
+      </div>
+      <div class="skills-div">
+        <ul class="competence-list">
+          <li class="list-title skills">
+            Skills
+          </li>
+        </ul>
+        <ul class="competence-list">
+          <li class="competence-item">Github</li>
+          <li class="competence-item">Codepen</li>
+          <li class="competence-item">Codesandbox</li>
+          <li class="competence-item">Replit</li>
+        </ul>
+      </div>
     </div>
   </section>
   <section class="contact">
```

Comment:
- [ ] Kindly use the proportional width for the paragraph on the left and the form on right hand side in the contact-me section as in the Figma design.
#### Design
![contact-me-parag-long-figma](https://user-images.githubusercontent.com/44057863/167276341-b67c0e69-2506-4e4d-b220-1486fb6bf859.png)
#### Imp
![contact-me-parag-long-impl](https://user-images.githubusercontent.com/44057863/167276345-f0f0b5ce-b482-421e-80da-abc1d8f03408.png)
l

Reply:
Done.